packages/seacas/libraries/ioss/src/gen_struc/Iogs_DatabaseIO.C

```diff
@@ -489,6 +489,18 @@ namespace Iogs {
     }
   }
 
+  std::vector<double> DatabaseIO::get_db_step_times_nl()
+  {
+    std::vector<double> timesteps;
+
+    int time_step_count = m_generatedMesh->timestep_count();
+    for (int i = 0; i < time_step_count; i++) {
```

Comment:
After line 496: `timesteps.reserve(time_step_count);` to avoid resizing/reallocations

Reply:
Ok, will do